docs/rollout/focus.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Focus Users
+layout: rollout
+---
+
+Release Tilt first to an initial set of Focus Users so you can give them a great experience.
+
+## Focus is Easier to Support than Spread Wide
```

Comment:
This is confusing construction; it reads like "it is easier to support focus than spread wide focus". Try "It's easier to support a narrow set of users than a broad one".